### PR TITLE
guest user adds product at its wishlist

### PR DESCRIPTION
**Pull request by tonicospinelli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Developing for Business
+=======================
+
+#### User Story
```
Comment (augustohp) [inline README.md:4]: This should be your commit message.

**Comment by augustohp** [comment]
Why 3 different commits? Each commit should be atomic, but hey are very small and as I see them they belong in the same _business value_ you want to achieve. along with being atomic, commits should be coherent as well.

**Comment by tonicospinelli** [comment]
@augustohp I understand your point of view, but I did 3 commits for showing application evolution from scratch.

I think my guilt has been added the wish list feature together with "applitcation bootsrap"

So, I'll take care in the next commits.

**Comment by tonicospinelli** [comment]
so, this PR should be added only 3th commit (800d570)

**Pull request merged**